packages/reakit/package.json

```diff
@@ -39,7 +39,8 @@
     "popper.js": "^1.14.4",
     "prop-types": "^15.6.2",
     "styled-components": "^3.4.2",
-    "styled-tools": "^1.1.0"
+    "styled-tools": "^1.1.0",
+    "reakit-theme-default": "^0.0.1"
```

Comment:
I used this and `lerna bootstrap` to make the package

Reply:
I don't think `reakit` should depend on `reakit-theme-default`.
`reakit` should be a peer dependency of `reakit-theme-default` instead.

Reply:
I mean. The guide said to do it, I'm aware it's not ideal. Since it worked I left it.